src/Microsoft.ML.TimeSeries/TimeSeriesProcessing.cs

```diff
@@ -2,17 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.ML.Runtime.EntryPoints;
-using Microsoft.ML.Runtime.TimeSeriesProcessing;
+using Microsoft.ML.EntryPoints;
+using Microsoft.ML.TimeSeriesProcessing;
 
-[assembly: EntryPointModule(typeof(TimeSeriesProcessing))]
+[assembly: EntryPointModule(typeof(TimeSeriesProcessingEntryPoints))]
```

Comment:
@ganik, @montebhoover: I assume NimbusML isn't taking in the TimesSeries work yet. If so, the renaming of the EntryPoint might affect NimbusML.